composer.json

```diff
@@ -120,7 +120,7 @@
         "cs-check": "phpcs",
         "cs-fix": "phpcbf",
         "serve": "php -S 0.0.0.0:8080 -t public/",
-        "static-analysis": "phpstan analyse",
+        "static-analysis": "phpstan analyse --memory-limit 256M",
```

Comment:
why not 1G instead of 256M ?

alternate command is to run phpstan directly 
`vendor/bin/phpstan analyse --memory-limit 1G
`